<launch>
    <node pkg="lidar_imu_calib" type="calib_exR_lidar2imu_node" name="calib_exR_lidar2imu_node" output="screen">
        <!-- real -->
        <param name="/data_type" value="real"/>
        <param name="/calib_type" value="double"/>  
        <param name="/bag_file" value="/home/xxiao/dataset/calib/small2_10.bag"/> 
        <param name="/chassis_topic" value="/chassis_msgs"/>   
        <param name="/lidar_topic" value="/velodyne_points"/>
        <param name="/imu_topic" value="/mti/sensor/imu"/>

        <!-- simulate -->
        <!-- <param name="/data_type" value="simulate"/>
        <param name="/calib_type" value="multi"/> -->
        <param name="/file_name1" value="/home/xxiao/dataset/calib/simulate/ex_0.0005time/1.txt"/>
        <param name="/file_name2" value="/home/xxiao/dataset/calib/simulate/ex_0.0005time/2.txt"/>
        <param name="/file_name3" value="/home/xxiao/dataset/calib/simulate/ex_0.0005time/3.txt"/>
    </node>
</launch>